homeassistant/components/sensor/rfxtrx.py

```diff
@@ -110,9 +111,9 @@ def __str__(self):
     @property
     def state(self):
         """Return the state of the sensor."""
-        if self.data_type:
+        if self.event:
             return self.event.values[self.data_type]
-        return None
+        return STATE_UNKNOWN
```

Comment:
The change from None to STATE_UNKOWN is not necessary, but I think it is more readable with UNKNOWN_STATE than None?
But before the method never returned None.